pkgs/applications/misc/limesuite/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, libusb, cmake, wxGTK30, mesa, libX11, soapysdr, sqlite, doxygen, gnuplot ? null }:
+
+stdenv.mkDerivation rec {
+  version = "17.12.0";
+  name = "LimeSuite-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/myriadrf/LimeSuite/archive/v${version}.tar.gz";
+    sha256 = "0fiadrkz5rh1yngjw6q6mg55k375pvp4ba9fkj3x8h3wh9ym32i6";
+  };
+
+  buildInputs = [ libusb cmake wxGTK30 mesa libX11 soapysdr sqlite doxygen gnuplot ];
+
+  configurePhase = ''
```

Comment:
Defining a configure phase is not necessary here. You can use `cmakeFlags = [ "-DENABLE_STREAM_UNITE=ON" ]` instead.